### DISCUSSION TITLE
Win11用最新版点击无反应

### DISCUSSION THREAD

**Discussion by jyuushi14** [getlantern/lantern-forum-cn#363] [Q&A]
Win11用最新7.8.3版本点击无反应，关机重启一样无反应

**Comment by MaxMKMax** [comment]
@jyuushi14 您好。好的，我会帮您转达给我们的负责人。抱歉，团队处于离线状态。他们将于今晚按照美国时区上线。我现在就通知他们，并帮助您今晚再次和他们跟进。会尽快帮您解决。抱歉让您久等了。谢谢!

非常抱歉影响了您的体验感。谢谢您的等待与支持。

您可以帮助我们在应用程序中提交报告您的问题吗？您可以访问Lantern应用程序并提交log/ 报告。

1. go to Support (转到支持)
2. select Report an Issue (选择报告问题)
3. choose issue type (选择问题类型)
4. include additional details (包括额外的详细信息)
5. hit Send Report (点击发送报告)

当您在app提交问题报告后您会收到邮件. 邮件标题上有 8 位数号码. 请回来这里留言您的log/ticket号码. 因为我需要拿您的反馈报告号码来和科技团队沟通您遇到的问题. 不然我无法提交您遇到的问题给科技团队.

有了这些资料，我们科技团队便可以尽快解决问题。您可以在留言区给我知道。谢谢。

**Comment by jyuushi14** [comment]
在哪一个应用程序中提交报告呢，如果是Lantern的话，现在也点击不进去的
